### PR TITLE
Cleanup colors in body texts and links

### PR DESCRIPTION
**Pull request by burakukula**
Together with Pencils, we decided to remove some of the text colors and bump main colors to the darker hex, for accessibility reasons. This is the first part of the cleanup that contains body text and links:

Expected results:
Screenshot 2019-04-18 at 14 46 49

**Implementation:**
Overview of the texts sizes, addtionaly I added font-size to the documentation, which might be useful:

Screenshot 2019-04-18 at 17 25 45

Screenshot 2019-04-18 at 15 40 32

Links overview:

Screenshot 2019-04-18 at 15 40 39

### CODE REVIEW

File: src/components/text/textConsts.js [hunk 1]
```diff
@@ -33,26 +30,31 @@ export const SIZE = TEXT_SIZE; // backward compatibility
...
+  GRAY_SECOONDARY: 'gray-secondary',
+  GRAY_SECOONDARY_LIGHT: 'gray-secondary-light'
 });
 
 export const COLOR = TEXT_COLOR; // backward compatibility
 
 export const TEXT_WEIGHT = Object.freeze({
   REGULAR: 'regular',
-  BOLD: 'bold',
-  EMPHASISED: 'bold' // backward compatibility
```
Comment (burakukula) [inline src/components/text/textConsts.js]: It's time to remove that one, it's not used any more.

File: src/components/text/textConsts.js [hunk 2]
```diff
@@ -17,9 +17,6 @@ export const TEXT_TYPE = Object.freeze({
 export const TYPE = TEXT_TYPE; // backward compatibility
 
 export const TEXT_SIZE = Object.freeze({
-  OBSCURE: 'xsmall', // backward compatibility
-  HEADLINE: 'large', // backward compatibility
-  STANDOUT: 'normal', // backward compatibility
```
Comment (burakukula) [inline src/components/text/textConsts.js]: They are not used any more

File: src/components/text/Link.jsx [hunk 3]
```diff
@@ -84,6 +84,7 @@ export const SIZE = {
   SMALL: 'small',
   OBSCURE: 'xsmall'
 };
+
 // backward compatibility
 export const COLOR = {
```
Comment (aju) [inline src/components/text/Link.jsx]: is this still needed ?

File: src/components/text/Link.jsx [hunk 4]
```diff
@@ -71,7 +71,7 @@ type TextPropsType = {
   className?: ?string
 };
 
-export const LINK_COLOR = TEXT_COLOR;
+export {LINK_COLOR} from './textConsts';
```
Comment (aju) [inline src/components/text/Link.jsx]: update flow type TextColorType ?

Comment (burakukula) [reply src/components/text/Link.jsx]: @aju done

File: src/components/text/pages/text.jsx [hunk 5]
```diff
@@ -7,68 +7,119 @@ import {TEXT_TYPE, TEXT_SIZE, TEXT_COLOR, TEXT_WEIGHT, TEXT_TRANSFORM, TEXT_ALIG
...
+];
+
 function getValues(object, addUndefined = true) {
   return addUndefined ? [undefined, ...Object.values(object)] : Object.values(object);
 }
 
 const TextExamples = () => {
   const SizeVariant = [];
-  const colorVariant = [];
+  const ColorVariant = [];
```
Comment (matzimowski) [inline src/components/text/pages/text.jsx]: `ColorVariant` => `colorVariant` ?
or is there any reason to start with capital letter?

Comment (burakukula) [reply src/components/text/pages/text.jsx]: @matzimowski no reason, I am a wierdoo 🙈

Comment (burakukula) [reply src/components/text/pages/text.jsx]: @matzimowski on my way to fix it.

**Pull request merged**